gui/GeometryInput.cpp

```diff
@@ -48,3 +54,37 @@ App * GeometryInput::handleEvent(SDL_Event event) {
 
 	return nextApp;
 }
+
+void GeometryInput::displayAngle() {
+	std::vector<Object*> params; // we need 2 lines to calc angle
+	params = this->parser->parseParameters(this->text);
+	Line *line1 = dynamic_cast<Line*>(params[0]);
+	Line *line2 = dynamic_cast<Line*>(params[1]);
+	double result = angle(*line1, *line2);
+	std::string resultString = this->funcResults->getText();
+	resultString += this->text;
+	resultString += ": ";
+	resultString += std::to_string(result);
```

Comment:
Limit length of number with `std::setprecision(2)`, a small extra step but worth it!
Maybe in #19

Reply:
Very good idea, will fix that